.travis.yml

```diff
@@ -1,21 +1,18 @@
-language: scala
+language: java
 sudo: false
 jdk:
   - oraclejdk8
 script:
-  - sbt +test
+  - mvn test -B
```

Comment:
You should be able to get travis to do both a Maven and an sbt build:

https://github.com/lagom/online-auction-java/pull/78/commits/55168c943c0eeb690673ba91931126c718d41c8f